lib/msal-browser/apiReview/msal-browser.api.md

```diff
@@ -1185,17 +1177,24 @@ export type LoadTokenOptions = {
 //
 // @public (undocumented)
 export class LocalStorage implements IWindowStorage<string> {
-    constructor();
+    constructor(clientId: string, logger: Logger, performanceClient: IPerformanceClient);
+    clear(): void;
     // (undocumented)
     containsKey(key: string): boolean;
     // (undocumented)
     getItem(key: string): string | null;
     // (undocumented)
     getKeys(): string[];
     // (undocumented)
+    getUserData(key: string): string | null;
```

Comment:
Do we want to call it `getAccountData()`?

Reply:
No, because this is used for more than just accounts. This should be used for anything that contains PII

Reply:
Ah that user! Got it. I have seen it used under getAccount, makes more sense now.